.pre-commit-config.yaml

```diff
@@ -19,6 +19,13 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
```

Comment:
so this won't take effect until we hit 0.910?

Reply:
That's a revision number for mypy, not our package. It tells pre-commit which version to fetch from their github releases

Reply:
thanks.  understood.